### DISCUSSION TITLE
Is there a way to enable link previews from signal-cli

### DISCUSSION THREAD

**Discussion by kidscissors-dylan** [AsamK/signal-cli#746] [Q&A]
I'm using Signal but don't have a smartphone, thanks to this awesome tool. However, there's one setting that I haven't been able to change from Signal Desktop: "Generate link previews" It says "To change this setting, open the Signal app on your mobile device and navigate to Settings > Chats"
Is there way to enable link previews from signal-cli?

**Comment by AsamK** [comment]
Changing the configurations for linked device is not yet implemented

**Comment by kidscissors-dylan** [comment]
cool thanks for responding. and thanks for this awesome tool! I use a dumb phone so its huge for me
